clients/fides-js/src/components/Overlay.tsx

```diff
@@ -54,23 +52,17 @@ const Overlay: FunctionComponent<Props> = ({
   const dispatchCloseEvent = useCallback(
     ({ saved = false }: { saved?: boolean }) => {
       dispatchFidesEvent("FidesModalClosed", cookie, options.debug, { saved });
-      if (!saved) {
-        onDismiss();
-      }
     },
     [cookie, options.debug]
   );
 
   const { instance, attributes } = useA11yDialog({
     id: "fides-modal",
-    role: window.Fides.options.preventDismissal ? "alertdialog" : "dialog",
```

Comment:
what did the `dialog` do vs `alertdialog` for the A11y component?

Reply:
this is for prevent dismissal because alertdialog cannot be dismissed. adding it back.